src/Nethermind/Nethermind.Core/Withdrawal.cs

```diff
@@ -26,16 +28,20 @@ public class Withdrawal
     public Address Address { get; set; } = Address.Zero;
 
     /// <summary>
-    /// Gets or sets the withdrawal amount in Wei.
+    /// Gets or sets the withdrawal amount in GWei.
     /// </summary>
-    public UInt256 Amount { get; set; }
+    [JsonProperty(PropertyName = "Amount")]
+    public ulong AmountInGWei { get; set; }
```

Comment:
I don't see any benefit in replacing `Amount` with `AmountInGWei` and `AmountInWei` besides code complication. There is a doc on top of the property already explaining it. If you're insisting on this change, then `GWei` should be changed to `Gwei`.

Why is the JSON field name capitalized?

Reply:
"Why is the JSON field name capitalized?" 
It doesn't matter for the library, but I can change it in our code.

"I don't see any benefit in replacing Amount with AmountInGWei and AmountInWei besides code complication"
@LukaszRozmej see benefits, I think I see some benefits too. I adjusted the code based on his feedback. I will change GWei to Gwei